scss/_modal.scss

```diff
@@ -50,6 +50,28 @@
   }
 }
 
+.modal-dialog-scrollable {
+  display: flex;
+  flex-direction: column;
+  height: calc(100% - (#{$modal-dialog-margin} * 2));
+  overflow: hidden;
+
+  > .modal-content {
```

Comment:
Only nitpick here is we likely don't need to use the `>` children selectors here—the HTML hierarchy is a requirement and we don't support nested modals. Thoughts?